lib/index.js

```diff
@@ -117,6 +117,11 @@ GA.prototype.initialize = function() {
     window.ga('require', 'displayfeatures');
   }
 
+  // check for enhanced link attribution
```

Comment:
You could, however, link to the part of the GA docs that cover enhanced link attribution